test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/CompositeModelBinderTest.cs

```diff
@@ -445,6 +445,7 @@ public class CompositeModelBinderTest
                 .Returns(
                     delegate (ModelBindingContext context)
                     {
+                        // Note this ModelBindingResult is not valid.
```

Comment:
With my latest changes, this test does not make sense. 
IMO lets replace it with `BindModel_DoesNotAddAValidationNode_IfModelBindingResultDoesNotContainAValidationNode`
PS: shorten the name if you want to.